src/main/java/org/jenkinsci/plugins/durabletask/PowershellScript.java

```diff
@@ -106,8 +117,8 @@ public String getScript() {
         // Copy the helper script from the resources directory into the workspace
         c.getPowerShellHelperFile(ws).copyFrom(getClass().getResource("powershellHelper.ps1"));
         
-        if (launcher.isUnix()) {
-            // There is no need to add a BOM with Open PowerShell
+        if (launcher.isUnix() || "pwsh".equals(powershellBinary)) {
```

Comment:
FYI: many launchers always say `true` and lie :-( (but this is existing issue) 

The binary is `pwsh.exe` (and on windows is normally case insensitive - so this should allows `pwsh` `pwsh.exe` and all cases of that if I understand the intent here)

Reply:
you do not have to specify `pwsh.exe`, `pwsh` is fine.

see existing code
https://github.com/jenkinsci/durable-task-plugin/blob/93b86c92fc0545cb4df79abe0ef2825a2c149827/src/main/java/org/jenkinsci/plugins/durabletask/PowershellScript.java#L86-L96

Reply:
you do not have to - but it is generally good practice to do so.
thus if you follow good practices then this will not work

Reply:
despite this being a String it is not currently user editable from jelly so I guess this is a moot point